maths/average_mode.py

```diff
@@ -1,7 +1,7 @@
 import statistics
 
 
-def mode(input_list):  # Defining function "mode."
+def mode(input_list: list) -> int:  # Defining function "mode."
     """This function returns the mode(Mode as in the measures of
     central tendency) of the input data.

```

Comment:
Now that a major bug was fixed, please add more tests. The following tests all have the resulting value be at the front of the list. This is the reason the bug was never caught. I suggest adding a test which does the mode of the following list, for example: `["x", "y", "y", "z"]`. This should return `"y"`.